api/src/main/java/org/openmrs/module/fhir/api/impl/MedicationServiceImpl.java

```diff
@@ -5,6 +5,7 @@
 import org.openmrs.module.fhir.api.MedicationService;
 import org.openmrs.module.fhir.api.db.FHIRDAO;
 import org.openmrs.module.fhir.api.strategies.medication.MedicationStrategyUtil;
+import org.springframework.stereotype.Component;
```

Comment:
why did you imported the component here?

Reply:
It is used in every Strategy and I didn't know about it. While I was working on FM-219 after adding a global property to config.xml I got an error that looked like "bean DefaultCohortStrategy not found". I added Component in FM-219 and this fixed an error. Then I realized that it is also missing in this branch so I added it.

Reply:
but this is service and it's not used here